scripts/zones/VeLugannon_Palace/mobs/Detector.lua

```diff
@@ -10,25 +10,51 @@ entity.onMobSpawn = function(mob)
     mob:setLocalVar("petCount", 1)
 end
 
+local getMobToSpawn = function(detector)
+    local ID = require("scripts/zones/VeLugannon_Palace/IDs")
```

Comment:
`local ID = require("scripts/zones/VeLugannon_Palace/IDs")`

This line should go above current line 5 `require("scripts/globals/regimes")`

Reply:
Will do.

Reply:
Moved it above line 5 as instructed.